lib/game.rb

```diff
@@ -0,0 +1,85 @@
+require "pry"
+class Game
+attr_accessor :board ,:player_1, :player_2
+
+def self.win_comb
+  WIN_COMBINATIONS
+end
+
+
+  WIN_COMBINATIONS =[
+   [0,1,2],
+   [3,4,5],
+   [6,7,8],
+   [0,3,6],
+   [1,4,7],
+   [2,5,8],
+   [0,4,8],
+   [2,4,6]
+ ]
+   def initialize(player_1=Players::Human.new("X"), player_2=Players::Human.new("O"), board=Board.new)#binding.pry
+     @board = []
```

Comment:
not sure why we are using this, but then declaring it again on 24. This is not needed.